manager/manager.go

```diff
@@ -623,3 +625,28 @@ func (self *manager) watchForNewContainers(quit chan error) error {
 	}()
 	return nil
 }
+
+func (self *manager) watchForNewOoms() error {
+	outStream := make(chan *oomparser.OomInstance)
+	oomLog, err := oomparser.New()
+	if err != nil {
+		return err
+	}
+	go func() {
+		err := oomLog.StreamOoms(outStream)
+		glog.Infof("Streaming ooms in manager")
+		if err != nil {
+			glog.Errorf("%v", err)
```

Comment:
Did you mean move the call to StreamOoms outside the Go routine here?  I made a change in oomparser so that that function StreamOoms will return rather than hang by putting its monitoring call in another goroutine.